### PR TITLE
GDM3 logon banner test and set

### PR DESCRIPTION
**Pull request by ghexp**

### CODE REVIEW

File: controls/V-219167.rb [hunk 1]
```diff
@@ -118,5 +118,20 @@
   tag fix_id: 'F-20891r304830_fix'
   tag cci: ['V-100561', 'SV-109665', 'CCI-000050']
   tag nist: ['AC-8 b']
-end
 
+  if !package('gdm3').installed?
```
Comment (aaronlippold) [inline controls/V-219167.rb:122]: please use `unless` rather than `if !`

**Review by aaronlippold — CHANGES_REQUESTED** [review]
Small change - also - take a look at what we do on the banner controls for RHEL7 where we strip out all the white space and other things which may be useful

File: controls/V-219167.rb (hunk 1 above)
Comment (aaronlippold) [reply controls/V-219167.rb:122]: Also, if you enable 'maintainers edit' for PRs I could help a bit as well

Comment (aaronlippold) [reply controls/V-219167.rb:122]: `unless package('gdm3').installed?`
    foo
  else
    bar
  end

Comment (ghexp) [reply controls/V-219167.rb]: Updated unless, brought in the gsub whitespace stripping from RHEL/CIS, and bumped threshold in [7089ab6](https://github.com/mitre/canonical-ubuntu-18.04-lts-stig-baseline/pull/24/commits/7089ab60ee17cd5fdb004327c387f09a6db2a745)

**Pull request merged**